src/org/sparkboard/server/server.clj

```diff
@@ -523,14 +543,14 @@
                                                 ("view_submission" "view_closed") (-> payload :view :callback_id)
                                                 "block_actions" (-> payload :actions first :action_id)))]
                       [handler-id (assoc context :slack/payload payload)]))]
-          (log/info handler-id (select-keys context [:slack/user-id
-                                                     :slack/team-id ]))
+          (log/info handler-id (select-keys context [:slack/user-id :slack/team-id ]))
           (log/debug :context context)
           (def LAST-CONTEXT context)
-          (binding [slack/*request* context]
-            ((handlers handler-id (fn [& args] (log/error :unhandled-request handler-id args)))
-             (assoc context ::handler-id handler-id)))))
-      (ring.http/ok))))
+          (.submit ^java.util.concurrent.ExecutorService pool
+                   ^Callable #(binding [slack/*request* context]
```

Comment:
Do we know how error handling differs b/w futures vs ExecutorService? (should we add error-handling/reporting here (like we have in `try-future`)?)

Reply:
Good question, I'll take a look

Reply:
Current situation is that exceptions inside the thread pool get *printed*, but do not throw or get logged. With the two commits I just pushed, we now use the snippet from Stuart Sierra to detect and properly *log* those exceptions. It shouldn't be a huge difference.

Note that these commits also change us from `.submit` onto the thread pool to `.execute` on the thread pool. This is a fire-and-forget approach that avoids wrapping the task in a Java Future, which as discussed is unnecessary work.